docs/grammar/SolidityLexer.g4

```diff
@@ -155,15 +155,20 @@ Not: '!';
 BitNot: '~';
 Inc: '++';
 Dec: '--';
+//@doc:inline
+DoubleQuote: '"';
+//@doc:inline
+SingleQuote: '\'';
 
 /**
- * A single quoted string literal restricted to printable characters.
+ * A single non-empty quoted string literal restricted to printable characters.
```

Comment:
Maybe it is better like this?
```suggestion
 * A single quoted non-empty string literal restricted to printable characters.
```

Reply:
Not sure - that might read like more like "single-quoted" :-)? Maybe I drop the "single" altogether...

Reply:
It looks much better to me now somehow :)

Reply:
I actually had to stop and check if the original meant `single and quoted` or `single-quoted` :) Dropping `single` sounds like the best solution to avoid ambiguity.